src/clr-angular/forms/select/options.spec.ts

```diff
@@ -31,6 +31,26 @@ export default function(): void {
             ifOpenService = context.getClarityProvider(IfOpenService);
         });
 
+        describe("Typescript API", function() {
+            it("has a host listener on mouseenter", () => {
```

Comment:
Nitpick: `"Has a host listener"` isn't descriptive. It'd be better with `"disables blur events on mouseenter"`.
Same comment for the next test, obviously.